Purchases/Public/Purchases.swift

```diff
@@ -908,8 +915,14 @@ public extension Purchases {
     @available(iOS 12.2, macOS 10.14.4, watchOS 6.2, macCatalyst 13.0, tvOS 12.2, *)
     @objc(purchasePackage:withDiscount:completionBlock:)
     func purchase(package: Package, discount: SKPaymentDiscount, completion: @escaping PurchaseCompletedBlock) {
-        let payment = storeKitWrapper.payment(withProduct: package.product, discount: discount)
-        purchase(product: package.product,
+        // todo: add support for SK2 with discounts, move to new class
```

Comment:
Same question, are you expecting to do this todo in this pr or later? If later, I think it's best if you'd create an issue and paste a link to it here, too.

Reply:
I am sorry @taquitos, these changes come from @aboedo's PR. I had to rebase on `swift_migration`. I will create the issue tho :)

Reply:
https://github.com/RevenueCat/purchases-ios/issues/848